header.cmake

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2008-2014 LAAS-CNRS, JRL AIST-CNRS.
+# Copyright (C) 2008-2014,2018 LAAS-CNRS, JRL AIST-CNRS, INRIA.
```

Comment:
Is it not more correct to update the copyright to `2008-2018` rather than having this gap? (note: I personally don't know)

Reply:
I've fixed it.